ipinfo/config.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/ipinfo/cli/lib"
+)
+
+type Config struct {
+	GlobalCache bool
+}
+
+// init function
+func InitConfig() error {
+	path, err := ConfigPath()
+	if err != nil {
+		return err
+	}
+	if !lib.FileExists(path) {
+		gConfig.GlobalCache = true
+		SetConfig(gConfig)
```

Comment:
So this logic is basically initializing the config to defaults if it doesn't exist right? I think we can make a function for atleast the defaults part separately, like:

```go
func NewConfig() Config {
	return Config{
		GlobalCache: true,
	}
}
```

Reply:
Also for these `SetConfig` you're currently not handling the returned error?